### PR TITLE
[KeyVault]  updating secret, key and certificate commands to take 'id' argument

### PR DESCRIPTION
**Pull request by schaabs**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by promptws** [comment]
View a preview at https://prompt.ws/r/Azure/azure-cli/6146
This is an **experimental preview** for @microsoft users.

File: src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/custom.py [hunk 1]
```diff
@@ -934,7 +936,8 @@ def delete_certificate_issuer_admin(client, vault_base_url, issuer_name, email):
 # region storage_account
 
 
-def backup_storage_account(client, vault_base_url, storage_account_name, file_path):
+def backup_storage_account(client, file_path, vault_base_url=None,
+                           storage_account_name=None, identifier=None):  # pylint: disable=unused-argument
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/custom.py]: I assume there's a reason for adding the unused arguments `identifier=None`

**Comment by derekbekoe** [comment]
Also, to fix CI, get the latest from dev branch (i.e. this commit - https://github.com/Azure/azure-cli/commit/e9d8d7ccb8b102b2aad09527309010358a4516c2)

**Comment by schaabs** [comment]
@tjprescott yes this fixes #3499, but we should probably leave it open until these changes are merged back into dev.

**Pull request merged**